Makefile.PL

```diff
@@ -61,6 +62,7 @@ my %WriteMakefileArgs = (
     "warnings" => 0
   },
   "TEST_REQUIRES" => {
+    "Data::Dumper" => 0,
```

Comment:
I am of two minds I may leave it in the test scripts though because I get tired of adding and removing it.

Reply:
Data::Dumper is a standard/core module.

You could add `skip = Data::Dumper` in the dist.ini under `[AutoPrereqs]`, and than it won't get added anymore.